scripts/legislation/africa/namibia_english.sh

```diff
@@ -13,23 +13,16 @@ function replace_parentheses_around_article_delimiters_with_square_brackets {
   sed -E 's/^\(([A-Za-z0-9]+)\)/[\1]/'
 }
 
-function add_newline_before_article_bodies {
-  local stdin="$(</dev/stdin)"
-
-  if [ "$#" -ne 1 ] ; then
-    echo_usage_error "$*" '<language>'
-    return 1
-  fi
-  local language="$1"
-
-  local article_regular_expression
-  article_regular_expression="$(get_article_regular_expression $language)"
-  local return_code="$?"
-  if [ "$return_code" -ne 0 ] ; then return "$return_code" ; fi
+function move_article_titles_after_article_numbers {
+  sed -En '${p;q};N;/\nArticle/{s/^(.*)\n(Article [0-9]+\.) /\2 \1|/p;b};P;D'
```

Comment:
If you want to understand how this works, check out [this](https://stackoverflow.com/a/4513283) Stack Overflow answer.